### DISCUSSION TITLE
offical orcaslicer discord

### DISCUSSION THREAD

**Discussion by ThePotatoLord014** [OrcaSlicer/OrcaSlicer#1040] [Ideas]
i really think we should make an official discord server if there is not one already.

**Comment by hliebscher** [comment]
https://discord.gg/TVgEDMyy at first step

**Reply by fr3ak2402** [reply]
The invitation link to the server has changed
https://discord.com/invite/EdZdH93W

The server is still under construction.

**Reply by liftbag** [reply]
> The invitation link to the server has changed https://discord.com/invite/EdZdH93W
> 
> The server is still under construction.

The invitation link is expired again.

**Reply by fr3ak2402** [reply]
Created a new link:
https://discord.gg/czxaUqJR

**Reply by liftbag** [reply]
> Created a new link: https://discord.gg/czxaUqJR

Thanks!

**Reply by quantumfrost** [reply]
> Created a new link: https://discord.gg/czxaUqJR

Expired again

**Reply by fr3ak2402** [reply]
Created a new link:
https://discord.gg/pjJ5tsNJBB

This should no longer expire.

**Reply by dvomang** [reply]
it expired

**Reply by ACK-Engineering** [reply]
Please send a new/permanent link. :-)

**Reply by peterek0000** [reply]
I found this one, created by SoftFever
https://discord.gg/P4VE9UY9gJ
